set.mm

```diff
@@ -319161,50 +319161,133 @@ can be the countable union of rare closed subsets (where rare means
       HZJSDGHZDKHQRSLQRTSABCDEFMNDOP $.
   $}
 
+  ${
+    cmslssbn.x $e |- X = ( W |`s U ) $.
+    ${
+      cmslssbn.s $e |- S = ( LSubSp ` W ) $.
+      $( Banach space property of a linear subspace: a linear subspace of a
+       normed vector space whose scalar field is a complete metric space is a
+       Banach space if the linear subspace is a complete metric space.
+       (Proposed by BJ, 7-Oct-2022.)  (Contributed by AV, 8-Oct-2022.) $)
```

Comment:
As with the theorems below, the part before the colon adds nothing (at best). Here, I would write:
```
$( A complete linear subspace of a normed vector space is a Banach space.  We furthermore
   have to assume that the field of scalars is complete since this is a requirement in the
   current definition of Banach spaces ~ df-ban .  (Contrib...
```
(I think that df-ban should probably be changed as I wrote in my previous comment, but this can be done later and then this theorem adapted later to remove that technical extra hypothesis.)
Also, you can remove "proposed by BJ".

Reply:
Done